pom.xml

```diff
@@ -62,6 +62,13 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-thymeleaf</artifactId>
 		</dependency>
+		
+		<dependency>
```

Comment:
please give proper format (indentation), also put the version in the properties section of POM file and use a placeholder here